WordPress/openverse
# Nuxt 3 robots.txt fixes

Adds a dynamically-generated robots.txt using the Nuxt SEO `robots` module. The module automatically prefixes robots.txt routes with our locale paths as well, which is nice. It is more verbose than the wildcard approach currently on https://openverse.org/robots.txt, but still works well.

It also automatically adds robots meta tags to pages based on these rules.

The module also automatically disables crawling in all non-production environments.  

## Testing instructions

- `ov j f` and visit localhost:8443/robots.txt to verify that the robotx.txt blocks all traffic in non-production setttings (as determined by the `site.indexable` property in the nuxt config.
- `ov j frontend/run build && ov j frontend/run start` and visit localhost:8443/robots.txt to verify that the production file contains the correct rules, including localized paths.
- Check that pages have the correct meta tags
